helix-core/src/main/java/org/apache/helix/tools/ClusterVerifiers/BestPossibleExternalViewVerifier.java

```diff
@@ -224,6 +224,9 @@ public boolean verifyByZkCallback(long timeout) {
   @Override
   protected synchronized boolean verifyState() {
     try {
+      if (_isLogMore) {
```

Comment:
I guess this helps your debug, but are you suggesting we check this into the master branch? In general, I don't like optional code in master.